ivy/functional/ivy/general.py

```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
     ret
         Boolean as to whether the input value is a nan or not.
 
+    Functional Examples
+    --------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([1])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([1])
+    >>> y = ivy.value_is_nan(x, False)
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([float('INF')])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    True
+
+    >>> x = ivy.array([float('INF')])
+    >>> y = ivy.value_is_nan(x, False)
+    >>> print(y)
+    False
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([1])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = ivy.native_array([1])
```

Comment:
Similarly you can remove the other 3 examples from here as well!

Reply:
Thank you for your comments. Removed .